apps/storybook.namekit.io/stories/Namekit/CurrencySymbol.stories.tsx

```diff
@@ -1,96 +1,90 @@
+} from "@namehash/namekit-react/client";
+import { CurrencyIconSize } from "@namehash/namekit-react/client";
 import type { Meta, StoryObj } from "@storybook/react";
 
-export const ETH: Story = {
-  args: {
-    currency: Currency.Eth,
-    size: CurrencySymbolSize.Large,
-  },
-};
-export const USD: Story = {
-  args: {
-    currency: Currency.Usd,
-    size: CurrencySymbolSize.Large,
-  },
-};
-export const USDC: Story = {
-  args: {
-    currency: Currency.Usdc,
-    size: CurrencySymbolSize.Large,
-  },
-};
-export const WETH: Story = {
-  args: {
-    currency: Currency.Weth,
-    size: CurrencySymbolSize.Large,
+const meta: Meta<typeof CurrencySymbol> = {
+  component: CurrencySymbol,
+  title: "Namekit/CurrencySymbol",
+  argTypes: {
+    currency: {
+      options: Object.values(Currency),
+      control: {
+        labels: Object.keys(Currency),
+        type: "select",
+      },
+    },
+    iconSize: {
+      options: Object.values(CurrencyIconSize),
+      if: { arg: "symbology", eq: CurrencySymbology.Icon },
+      control: {
+        labels: {
+          [CurrencyIconSize.Small]: "Small (16px)",
+          [CurrencyIconSize.Large]: "Large (20px)",
+        },
+        type: "select",
+      },
+    },
+    className: {
+      if: { arg: "symbology", neq: CurrencySymbology.Icon },
+      control: {
+        type: "text",
+      },
+    },
+    describeCurrencyInTooltip: { control: { type: "boolean" } },
+    symbology: {
+      options: Object.keys(CurrencySymbology),
+      control: { type: "select" },
+    },
   },
-};
-export const DAI: Story = {
   args: {
-    currency: Currency.Dai,
-    size: CurrencySymbolSize.Large,
+    iconSize: CurrencyIconSize.Small,
+    describeCurrencyInTooltip: true,
+    symbology: CurrencySymbology.TextSymbol,
   },
 };
-export const SmallSize: Story = {
+
+export default meta;
+
+type Story = StoryObj<typeof CurrencySymbol>;
+
+export const AsAnAcronym: Story = {
   args: {
-    size: CurrencySymbolSize.Small,
     currency: Currency.Eth,
+    symbology: CurrencySymbology.Acronym,
   },
 };
-export const LargeSize: Story = {
+export const AsAnIcon: Story = {
   args: {
-    size: CurrencySymbolSize.Large,
     currency: Currency.Eth,
+    symbology: CurrencySymbology.Icon,
   },
 };
-export const WithCustomSymbolColor: Story = {
+export const WithCustomIconColor: Story = {
+  argTypes: {
+    fill: { control: { type: "color" } },
+  },
   args: {
-    symbolFillColor: "#007bff",
-    size: CurrencySymbolSize.Large,
     currency: Currency.Eth,
+    symbology: CurrencySymbology.Icon,
+    fill: "#007bff",
   },
 };
-export const ShowingTooltipDescription: Story = {
+export const WithCustomFontStyle: Story = {
+  argTypes: {
+    className: { control: { type: "text" } },
+  },
   args: {
-    describeCurrencyInTooltip: true,
-    size: CurrencySymbolSize.Large,
     currency: Currency.Eth,
+    className: "nk-text-3xl colorful-text",
   },
 };
 export const NotShowingTooltipDescription: Story = {
```

Comment:
This story should be inverted now that the default is `false`. Therefore we now need a story showing the `true` case. Please be sure to update the name of the story too.